format/Message.fbs

```diff
@@ -64,8 +64,11 @@ table Date {
 table Time {
 }
 
+enum TimeUnit: short { MILLISECOND, MICROSECOND }
```

Comment:
That's fine to drop the metadata in those cases when writing to Parquet. You would need to write key-value metadata to be able to recover the exact metadata (timestamp in nanos)